cms/djangoapps/contentstore/views/course.py

```diff
@@ -642,6 +642,8 @@ def _create_or_rerun_course(request):
             fields['display_name'] = display_name
 
         if 'source_course_key' in request.json:
+            fields['advertised_start'] = None
```

Comment:
@AkA84 yeah, I'm hoping someone on the Platform team can chime in here for a better way to do this. @alawibaba ?